src/metabase/models/field_values.clj

```diff
@@ -27,6 +27,7 @@
    [java-time.api :as t]
    [malli.core :as mc]
    [medley.core :as m]
+   [metabase.analyze :as analyze]
```

Comment:
This dependency bothers me a bit, in my mind models shouldn't depend on higher level modules, if possible. We could "fix" this by moving the constants out to a general config namespace at some point.

Reply:
agreed, but that's a step 2 thing and step 1 is just getting things in modules. It's going to take a while and several iterations to clean everything up